### PR TITLE
V2 cleanup

### PR DESCRIPTION
**Pull request by LorbusChris**
/cc @bgilbert @yussufsh

### CODE REVIEW

File: website/docs/d/config.html.md [hunk 1]
```diff
@@ -44,12 +44,11 @@ The following arguments are supported:
...
 * `groups` - (Optional) The list of groups to be added.
 
-* `append` - (Optional) Any number of blocks with the configs to be appended to the current config.
+* `merge` - (Optional) A list of the configs to be merged to the current config.
 
 * `replace` - (Optional) A block with config that will replace the current.
 
-
-The `append` and `replace` blocks supports:
+The `merge` and `replace` blocks supports:
```
Comment (bgilbert) [inline website/docs/d/config.html.md]: s/supports/support/

**Review by bgilbert — APPROVED** [review]
LGTM.  We don't intend to record changes anywhere?

**Comment by LorbusChris** [comment]
@bgilbert we haven't been updating it and I feel like it's overhead. Also there's always the git history.
I can drop the removal if you think it's still useful.

**Comment by bgilbert** [comment]
I'd be okay with e.g. switching to having release notes in the GitHub release.  But IMO projects should generally have some sort of user-visible release notes that are more readable than the Git history.

**Comment by LorbusChris** [comment]
I'd prefer the notes on the GH releases. I'll leave the decision up to you :)
Feel free to hit merge or ping me again and I'll drop the commit.

**Pull request merged**